forge/tests/it/repros.rs

```diff
@@ -183,12 +183,6 @@ fn test_issue_3347() {
     );
 }
 
-// <https://github.com/foundry-rs/foundry/issues/3616>
-#[test]
-fn test_issue_3616() {
-    test_repro!("Issue3616");
-}
-
```

Comment:
no longer applicable since the test contract address is now different with no balance in any chain (0x5615dEB798BB3E4dFa0139dFa1b3D433Cc23b72f)